defaults/main.yml

```
@@ -30,8 +30,9 @@ msmtp_default_account: yandex
 msmtp_domain:          yandex.com
 
 ## Logging (Select A) "syslog" or B) "file" logging or C) "no" log
-msmtp_log:       file
-msmtp_logfile:   ~/.msmtp.log
+msmtp_log:             file
+msmtp_syslog_facility: ""
```

Setting this to an empty string would change [the existing default `LOG_USER`](https://marlam.de/msmtp/msmtp.html#Commands-specific-to-sendmail-mode), which would be a bad idea:
> The default is ‘LOG_USER’.

Let's set it to that:

```suggestion
+## If using `syslog`, can be one of 'LOG_USER', 'LOG_MAIL', 'LOG_LOCAL0', …, 'LOG_LOCAL7'.
+msmtp_syslog_facility: "LOG_USER"
```